.devcontainer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.231.3/containers/python-3/.devcontainer/base.Dockerfile
```

Comment:
Could you extend the README.md with a small paragraph about how this is supported for development?

Reply:
Absolutely. Will add it.

Reply:
done